sound/soc/sof/amd/acp.h

```diff
@@ -82,6 +83,8 @@
 #define SRAM1_SIZE				0x13A000
 #define PROBE_STATUS_BIT			BIT(31)
 
+#define FW_SIGNATURE				0x100
```

Comment:
prefix? FW_ is awfully generic.

Reply:
Okay, will change it to FIRMWARE_SIGNATURE